internal/pkg/manifests/receive/builder.go

```diff
@@ -555,6 +693,26 @@ func ingestorArgsFrom(opts IngesterOptions) []string {
 	return args
 }
 
+func routerArgsFrom(opts RouterOptions) []string {
+	opts.Options = opts.ApplyDefaults()
+	args := []string{
+		"receive",
+		fmt.Sprintf("--log.level=%s", *opts.LogLevel),
+		fmt.Sprintf("--log.format=%s", *opts.LogFormat),
+		fmt.Sprintf("--grpc-address=0.0.0.0:%d", GRPCPort),
+		fmt.Sprintf("--http-address=0.0.0.0:%d", HTTPPort),
+		fmt.Sprintf("--remote-write.address=0.0.0.0:%d", RemoteWritePort),
+		fmt.Sprintf("--receive.replication-factor=%d", opts.ReplicationFactor),
+		fmt.Sprintf("--receive.hashrings-algorithm=%s", AlgorithmKetama),
+		fmt.Sprintf("--receive.hashrings-file=%s/%s", hashringMountPath, HashringConfigKey),
```

Comment:
Let's add an option field for receive.hashrings-file-refresh-interval as well, would be good for quick tests